components/quiz/Quiz.tsx

```diff
@@ -12,10 +13,25 @@ export type Answer = {
 };
 
 function Quiz({ question, answerOptions }: QuizProps) {
+  const handleAnswerClick = (isCorrect: boolean) => {
+    if (isCorrect === true) {
+      alert("AMAZING"), setColorchange("green");
```

Comment:
This is an uncommon code style.

```suggestion
      alert("AMAZING");
      setColorchange("green");
```